src/main/java/Luke.java

```diff
@@ -0,0 +1,128 @@
+import java.io.*;
```

Comment:
Perhaps the relevant `io` files could be listed and imported explicitly instead of using a wildcard import?

Reply:
I'll specify the files to abide by the coding standards.